driver/tke_driver.go

```diff
@@ -7,29 +7,33 @@ import (
 	"strings"
 	"time"
 
-	"github.com/rancher/kontainer-engine-driver-tencent/tencentcloud/ccs"
 	"github.com/rancher/kontainer-engine/drivers/options"
 	"github.com/rancher/kontainer-engine/drivers/util"
 	"github.com/rancher/kontainer-engine/types"
 	"github.com/rancher/rke/log"
 	"github.com/sirupsen/logrus"
 	tccommon "github.com/tencentcloud/tencentcloud-sdk-go/tencentcloud/common"
-	tcerrors "github.com/tencentcloud/tencentcloud-sdk-go/tencentcloud/common/errors"
 	"github.com/tencentcloud/tencentcloud-sdk-go/tencentcloud/common/profile"
+	cvm "github.com/tencentcloud/tencentcloud-sdk-go/tencentcloud/cvm/v20170312"
+	tke "github.com/tencentcloud/tencentcloud-sdk-go/tencentcloud/tke/v20180525"
 	"golang.org/x/net/context"
 
 	"k8s.io/client-go/kubernetes"
 	"k8s.io/client-go/rest"
 )
 
 const (
-	runningStatus        = "Running"
-	successStatus        = "Success"
-	failedStatus         = "Failed"
-	notReadyStatus       = "ClusterNotReadyError"
-	processRunningStatus = "ProcessAlreadyRunning"
-	retries              = 5
-	pollInterval         = 30
+	runningStatus  = "Running"
+	successStatus  = "Created"
+	failedStatus   = "CreateFailed"
+	notReadyStatus = "ClusterNotReadyError"
+	retries        = 5
+	pollInterval   = 30
+)
+
+var (
+	terminate      = "terminate"
+	managedCluster = "MANAGED_CLUSTER"
```

Comment:
Rename the variables to `instanceDeleteModeTerminate` and `managedClusterType` to help people know what they are used for.